Arduino_Sub_DAQ/SiMlx.ino

```diff
@@ -0,0 +1,75 @@
+/*************************************************** 
+  This is a library example for the MLX90614 Temp Sensor
+
+  Designed specifically to work with the MLX90614 sensors in the
+  adafruit shop
+  ----> https://www.adafruit.com/products/1748
+  ----> https://www.adafruit.com/products/1749
+
+  These sensors use I2C to communicate, 2 pins are required to  
+  interface
+  Adafruit invests time and resources providing this open source code, 
+  please support Adafruit and open-source hardware by purchasing 
+  products from Adafruit!
+
+  Written by Limor Fried/Ladyada for Adafruit Industries.  
+  BSD license, all text above must be included in any redistribution
+ ****************************************************/
+
+#include <Wire.h>
+#include <Adafruit_MLX90614.h>
+#define IR1 0x5C
+#define IR2 0x5A
+#define IR3 0x5B
+#define IR4 0x5D
+Adafruit_MLX90614 mlx;
```

Comment:
Ditto. How about, `temp_sensor`?

Reply:
Did you mean that I should change 'Adafruit_MLX90614 mlx;' to 'temp_sensor;'?
If so, output is Compilation error: 'temp_sensor' does not name a type

Reply:
Change the name of the variable, which is currently `mlx`.
`Adafruit_MLX90614` refers to the name of its corresponding class. It's like "int" when declaring an integer variable, `int x`